apps/hash-api/src/generate-ontology-type-ids.ts

```diff
@@ -7,6 +7,11 @@ import { createGraphClient } from "@local/hash-backend-utils/create-graph-client
 import { getRequiredEnv } from "@local/hash-backend-utils/environment";
 import { Logger } from "@local/hash-backend-utils/logger";
 import { publicUserAccountId } from "@local/hash-backend-utils/public-user-account-id";
+import type {
+  DataType,
+  EntityType,
+  PropertyType,
+} from "@local/hash-graph-client";
```

Comment:
We can import these from `@blockprotocol/type-system`

Reply:
We can, but does it matter? We import them inconsistently across the codebase, is there a difference?

We should enforce a specific import if there's a reason we shouldn't import them from here

Reply:
We will make this consistent, I will separately add an eslint rule